### DISCUSSION TITLE
Create a wiki page for suggested additional APKs

### DISCUSSION THREAD

**Discussion by FlattusBlastus** [nextcloud/all-in-one#2036] [Questions] [answered]
It's hard to know what has been included at this point. Knowing what's already been included, are there any suggestions on what to add for a new install?

**Comment by szaimen** [comment, marked as answer]
Hi, you can have a look at this: https://github.com/nextcloud/all-in-one/blob/main/Containers/nextcloud/Dockerfile to see what is included already.

**Comment by FlattusBlastus** [comment]
So at the time of this post, the same as Zoey2936's:
-e NEXTCLOUD_ADDITIONAL_APKS="imagemagick aria2 yt-dlp ocrmypdf tesseract-ocr-data-deu sane-utils netpbm"

**Reply by Zoey2936** [reply]
what do you mean with this exactly? I'm a bit confused.
